tests/app/celery/test_broadcast_message_tasks.py

```diff
@@ -47,10 +46,9 @@ def test_send_broadcast_event_only_sends_to_one_provider_if_set_on_service(
     notify_api,
     sample_service
 ):
-    notify_db.session.add(ServiceBroadcastProviderRestriction(
-        service=sample_service,
-        provider='vodafone'
-    ))
+    settings = ServiceBroadcastSettings(service=sample_service, channel="test", provider="vodafone")
+    settings.provider = "vodafone"
```

Comment:
dont think we need second line here

```suggestion
    settings = ServiceBroadcastSettings(service=sample_service, channel="test", provider="vodafone")
```